config/settings/development.yml

```diff
@@ -0,0 +1,36 @@
+:log:
+  :collection:
+    :archive:
+      :pattern:
+    :current:
+      :pattern:
+      - log/*.log
+      - log/apache/*.log
+      - log/*.txt
+      - config/*
+      - "/var/log/syslog*"
+      - "/var/log/daemon.log*"
+      - "/etc/default/ntp*"
+      - "/var/log/messages*"
+      - "/var/log/cron*"
+      - BUILD
+      - GUID
+      - VERSION
+    :ping_depot: true
+    :ping_depot_timeout: 20
+  :level: debug
+  :level_rails: debug
```

Comment:
:+1:

On Tue, Mar 29, 2016 at 10:54 AM, Jason Frey <notifications@github.com>
wrote:

> In config/settings/development.yml
> <https://github.com/ManageIQ/manageiq/pull/7569#discussion_r57737140>:
>
> > +      - log/*.log
> > +      - log/apache/*.log
> > +      - log/*.txt
> > +      - config/*
> > +      - "/var/log/syslog*"
> > +      - "/var/log/daemon.log*"
> > +      - "/etc/default/ntp*"
> > +      - "/var/log/messages*"
> > +      - "/var/log/cron*"
> > +      - BUILD
> > +      - GUID
> > +      - VERSION
> > +    :ping_depot: true
> > +    :ping_depot_timeout: 20
> > +  :level: debug
> > +  :level_rails: debug
>
> i.e. I think all you need is
>
> ---
> :log
>   :level_rails: debug
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/7569/files/b440d6ac8a4b263a00cbd21b7d1ee6b5ef60630a#r57737140>
>



-- 
Chris Arcand
@chrisarcand